docs/src/image_loading.md

```diff
@@ -0,0 +1,203 @@
+# Image Loading
+
+## Overview
+
+The Image Loading module is a component of the MCU Runtime SDK designed for managing SOC images stored in flash memory. This module provides APIs for:
+
+- Detecting SOC images availability in the [flash storage](./image_loading.md).
+- Parsing the [SOC manifest](https://github.com/chipsalliance/caliptra-sw/blob/main/auth-manifest/README.md).
+- Loading SOC images to a target component.
+
+The diagram below illustrates an **example** how the Caliptra subsystem, integrated within the SOC, is used to load SOC images to vendor components. In this example, these are the 2 Vendor CPUs and their corresponding RAMs.
+
+<p align="center">
+    <img src="images/image_loading_sample.svg" alt="flash_config" width="50%">
```

Comment:
Could we get this to match the style a bit closer to what is in the other documents (the other diagram is closer)? I can help with this if you provide me the source.

But specifically:

* Text should be larger so that is legible (it is currently a bit small)
* No gradients, just use simple, soft colors
* No white text on a dark background

Reply:
Updated the image. You can find the raw image in page-10 of this Visio file: https://microsoft-my.sharepoint.com/:u:/p/marcovisaya/EcaNju7LbulCjYiP533qQzIB2ZruXr3RtT_XBiOwlOCA9Q?e=DAdazr